index.html

```diff
@@ -43,5 +43,134 @@ <h1 class="about__title">I’m Mariane <br> Glad to see <br> you!</h1>
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+        </ul>
+        <button class="see__project">See Project</button>
+    </div>
+    <div class="works__item tonic2__project">
+        <div class="tonic2__header"></div>
+        <h3 class="project__title">Tonic</h3>
+        <div class="project__tags">
+            <p>CANOPY</p>
+            <div class="project__tags-boulet gray_color"></div>
+            <div class="gray_color">Back End Dev</div>
+            <div class="project__tags-boulet gray_color"></div>
+            <div class="gray_color">2015</div>
+        </div>
+        <p class="project__description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="project__languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+        </ul>
+        <button class="see__project">See Project</button>
+    </div>
+    <div class="works__item multiPost2__project">
+        <div class="multi2__header"></div>
+        <h3 class="project__title">Multi-Post Stories</h3>
+        <div class="project__tags">
+            <p>CANOPY</p>
+            <div class="project__tags-boulet gray_color"></div>
+            <div class="gray_color">Back End Dev</div>
+            <div class="project__tags-boulet gray_color"></div>
+            <div class="gray_color">2015</div>
+        </div>
+        <p class="project__description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="project__languages">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+        </ul>
+        <button class="see__project">See Project</button>
+    </div>    
+  </section>
+  <section class="about__mySelf">
+    <div class="about__div1">
+        <h2 class="mySelf__title">About <br> Myself</h2>
+    <p class="about_paragraph">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p class="about__links">LET'S CONNECT</p>
+    <ul class="about__links__icons flex">
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+        <li><a href="http://github.com/Mukaba/" target="_blank" rel="noopener noreferrer"></a></li>
+    </ul>
+    <button class="get__resume">Get my resume</button>
+    </div>
+    <div class="about__div2">
+        <div class="languages_title">
```

Comment:
- [ ] kindly make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li